File: Python/program95.py
# Input : 4
# Output :
'''
    1 * 3 *
    1 * 3 *
    1 * 3 *
    1 * 3 *
'''
class Pattern:

    def __init__(self,iRow,iCol):
        self.Row = iRow
        self.Col = iCol
            
    def Display(self):
        for i in range(1,self.Row + 1):
            for j in range(1,self.Col + 1):
                if((j % 2) == 0):
                    print("*\t", end="")
                else:
                    print(j, "\t", end="")
            print()
            
def main():
    print("Enter number of rows ")
    iValue1 = int(input())
        
    print("Enter number of columns ")
    iValue2 = int(input())
    
    pobj = Pattern(iValue1,iValue2)
        
    pobj.Display()
   
if __name__ == "__main__":
    main()

File: C++/program95.cpp

/*
    1 2 3  4
    5 6 7  8
    9 10 11 12

*/

#include<iostream>
using namespace std;

class Pattern
{
    private:
        int iRow;
        int iCol;
    public:
        Pattern(int Row, int Col)
        {
            iRow = Row;
            iCol = Col;
        }

        void Display()
        {
            int j = 0, i = 0,icnt = 1;;
        
            for(i = 1 ; i <= iRow; i++)
            {
                for(j = 1; j <= iCol; j++,icnt++)
                {
                    cout<<icnt<<'\t';
                }
                cout<<'\n';
            }

        }
};

int main()
{

    int iValue1 = 0, iValue2 = 0;

    cout<<"Enter rows "<<'\n';
    cin>>iValue1;

    cout<<"Enter column "<<'\n';
    cin>>iValue2;

    Pattern pobj(iValue1,iValue2);
    pobj.Display();

    return 0;
}